ql/src/java/org/apache/hadoop/hive/ql/parse/repl/CopyUtils.java

```diff
@@ -429,7 +429,7 @@ public void doCopy(Path destination, List<Path> srcPaths) throws IOException, Lo
   */
   boolean regularCopy(FileSystem sourceFs, List<ReplChangeManager.FileInfo> fileList)
       throws IOException {
-    if (hiveInTest) {
+    if (hiveInReplTest) {
```

Comment:
do we have test for hiveInReplTest = false

Reply:
Do we require that hiveInReplTest = false cases?